_includes/events-hero.html

```diff
@@ -0,0 +1,40 @@
+{% macro render(hero) %}
+
+{% from "post-macros.html" import category_icon as category_icon %}
+
+<section class="content_hero hero">
+    <div class="hero_wrapper">
+        <a href="#" class="hero_card hero_card__events">
```

Comment:
@Scotchester Good point, you're right, I neglected to account for the outer `section`. As far as I know single sub-sections within sections are fine, but only really make sense when the parent section has heading content, otherwise they lead to untitled headings in the outline, e.g.:

![screen shot 2015-02-17 at 5 52 13 pm](https://cloud.githubusercontent.com/assets/704760/6249284/2f33b8c0-b754-11e4-98ef-b33332ac1e61.png)

I would still consider incorporating a `header` to organize all the hero's intro markup together (and possibly a `footer`—though based on @jimmynotjim clarification the event date, it's probably more applicable as body content). However, as @jimmynotjim pointed out structural changes to the pattern are probably out of scope for this PR.

@jimmynotjim That all makes sense.